assets/README

```diff
@@ -504,6 +537,15 @@ If Tango is properly installed at your site, a copy of the documentation should
 <install_prefix>/share/doc/tango
 You can also look at the Tango web site at https://www.tango-controls.org
 
+Sphinx is used to generate the tango documentation html pages.
+Nevertheless, if you want to re-generate the documentation, you can use 
+the command "make html" in doc/src directory.
+This requires Sphinx and IPython to be installed on your host.
+On a Debian-like system, you can install these tools using the following 
+commands:
+sudo apt install -y python python-pip
+pip install Sphinx ipython
+
```

Comment:
`Sphinx -> sphinx` in the pip command. Is ipython really a requirement? I might be convenient to use but I don't think it is strictly required.

Reply:
As for the IPython, it is necessary to build the docs:

https://github.com/tango-controls/tango-doc/blob/982785add4e1e1d497106bdc39f5ced65990d57c/requirements.txt#L3

https://github.com/tango-controls/tango-doc/blob/982785add4e1e1d497106bdc39f5ced65990d57c/source/conf.py#L19

Reply:
I've fixed the `sphinx` casing.